package.json

```diff
@@ -1,110 +1,114 @@
 {
-    "name": "clarity-all",
```

Comment:
It looks like the editor formatted the entire file. Can you revert this so that only the new packages installed are reflected in the the commit?